### PR TITLE
Package crontab.0.1

### PR DESCRIPTION
**Pull request by yurug**
### `crontab.0.1`
Interacting with cron from OCaml
With this library, an ocaml program can interact with cron:
parse/print crontab and update installed crontabs.

---
* Homepage: https://github.com/yurug/ocaml-crontab
* Source repo: git://github.com/yurug/ocaml-crontab.git
* Bug tracker: https://github.com/yurug/ocaml-crontab/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/crontab/crontab.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+homepage: "https://github.com/yurug/ocaml-crontab"
+bug-reports: "https://github.com/yurug/ocaml-crontab/issues"
+dev-repo: "git://github.com/yurug/ocaml-crontab.git"
+
+depends: [
+  "dune"                 {build & >= "1.4.0"}
+  "ocaml"
+  "odoc"                 {with-doc}
+  "ocamlfind"
```
Comment (mseri) [inline packages/crontab/crontab.0.1/opam]: You don't seem to be using ocamlfind in the library and tests code, only in the example when running as a script.

Comment (yurug) [reply packages/crontab/crontab.0.1/opam]: You are right. Let's minimize the dependencies.

Comment (yurug) [reply packages/crontab/crontab.0.1/opam]: Thanks

**Comment by mseri** [comment]
Thanks

**Pull request merged**